status-variables.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 服务器状态变量
+summary: 使用状态变量查看系统和会话状态
+---
+
+# 服务器状态变量
+
+服务器状态变量提供有关服务器全局状态和 TiDB 中当前会话状态的信息。大多数变量与 MySQL 兼容。
+
+你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令检索全局状态, 当前会话状态可以使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令。 
+
+此外, [FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) 命令与 MySQL 兼容。
+
+## 系统变量
+
+### Compression
+
+- 作用域: SESSION
```

Comment:
Global comment: we should use *full-width* colon for Chinese text. Please update the rest of the document.
```suggestion
- 作用域：SESSION
```

Reply:
Done